tutorretirement/patches/k8s-jobs

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: retirement-job
+  labels:
+    app.kubernetes.io/component: cronjob
```

Comment:
Correct to assume that the other labels that Tutor sets will come from `kustomize`?

Reply:
I don't completely understand what this line does, tbh. I thought it was a selector. I saw that Jobs have this. Do you mean that `cronjob` needs to be defined somewhere?

Reply:
No, having checked on what Jobs look like this is probably fine. :)